pkgs/applications/misc/rofi-top/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, cairo
+, glib
+, gobject-introspection
+, libgtop
+, pkg-config
+, rofi-unwrapped
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rofi-top";
+  version = "unstable-2017-10-16";
+
+  src = fetchFromGitHub {
+    owner = "davatorium";
+    repo = pname;
+    rev = "9416addf91dd1bd25dfd5a8c5f1c7297c444408e";
+    sha256 = "sha256-lNsmx1xirepITpUD30vpcs5slAQYQcvDW8FkA2K9JtU=";
+  };
+
```

```suggestion

  patches = [
    ./0001-Patch-plugindir-to-output.patch
    ./0002-Patch-add-cairo.patch
  ];

```

can we upstream this?

---

Sure, I'll try for the cairo patch.

---

Got an answer for cairo from the author. Removing this dependency is on his todolist however I don't have an ETA so I think we shall keep the patches for the moment.